content/articles/2019-09-tidymodels.Rmarkdown

```diff
@@ -0,0 +1,140 @@
+
+### Improvements
+
+* `correlate()`'s interface for databases was improved. It now only calculates unique pairs, and simplifies the formula that ultimately runs in-database. We also re-added the vignette to the package, which is also available in site as an [article](https://tidymodels.github.io/corrr/articles/databases.html)
+
+## tidypredict 0.4.3
+
+### New models
+
+The new version is now able to parse the following models:
+
+  * `cubist()`, from the `Cubist` package
+  * `ctree()`, from the `partykit` package
+  * XGBoost trained models, via the `xgboost` package
+
+### New features
+
+* Integration with `broom`'s `tidy()` function. It works with Regression models only
+* Adds support for `parsnip` fitted models: `lm`, `randomForest`, `ranger`, and `earth`
+* Adds [`as_parsed_model()`](https://tidymodels.github.io/tidypredict/reference/as_parsed_model.html) function. It adds the proper class components to the list. This allows any model exported in the correct spec to be read in by `tidypredict`. See the [Save Models](https://tidymodels.github.io/tidypredict/articles/save.html) and [Non-R models](https://tidymodels.github.io/tidypredict/articles/non-r.html) for more information
+
+### Improvements
+
+* Now supports classification models from `ranger`
+
+### Website
+
+The package's [official website](https://tidymodels.github.io/tidypredict/index.html) has been expanded greatly. Here are some highlights:
+
+  * An article per each supported model, they are found under Model List
+  * A how to guide to save and reload models, [link here](https://tidymodels.github.io/tidypredict/articles/save.html) 
+  * How to integrate non-R models to `tidypredict`, [link here](https://tidymodels.github.io/tidypredict/articles/non-r.html) 
+
+## yardstick 0.0.4
+
+### New Metrics
+
+Two new metrics have been added to yardstick:
+
+* [`iic()`](https://tidymodels.github.io/yardstick/reference/iic.html) is a numeric metric for computing the index of ideality of correlation. It is a potential alternative to the traditional correlation coefficient, and has been used in QSAR models ([#115](https://github.com/tidymodels/yardstick/issues/87)).
+* [`average_precision()`](https://tidymodels.github.io/yardstick/reference/average_precision.html) is a probability metric that can be used as an alternative to `pr_auc()`. It has the benefit of avoiding any issues of ambiguity in the edge case where `recall == 0` and the current number of false positives is `0`.
+
+### Improvements
+
+* `pr_curve()` (and by extension `pr_auc()`) have been greatly improved to better handle edge cases when duplicate class probability values are present. Additionally, the first precision value in the curve is now a `1`, rather than an `NA`, which results in a more practical curve, and generates a more correct AUC value ([#93](https://github.com/tidymodels/yardstick/issues/93)).
```

Comment:
```suggestion
* `pr_curve()` (and by extension `pr_auc()`) has been greatly improved to better handle edge cases when duplicate class probability values are present. Additionally, the first precision value in the curve is now a `1`, rather than an `NA`, which results in a more practical curve, and generates a more correct AUC value ([#93](https://github.com/tidymodels/yardstick/issues/93)).
```